hack/ginkgo-e2e.sh

```diff
@@ -127,6 +127,7 @@ export PATH=$(dirname "${e2e_test}"):"${PATH}"
   --repo-root="${KUBE_ROOT}" \
   --node-instance-group="${NODE_INSTANCE_GROUP:-}" \
   --prefix="${KUBE_GCE_INSTANCE_PREFIX:-e2e}" \
+  --network="${NETWORK:-e2e}" \
```

Comment:
I chose network because of: https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-test.sh#L54, https://github.com/kubernetes/kubernetes/blob/master/cluster/gke/config-test.sh#L19, recommendations?